.yarnrc

```diff
@@ -0,0 +1 @@
+yarn-offline-mirror "../kibana-extra/kibana-offline-mirror"
```

Comment:
probably should replace this with a ci only config

Reply:
We could use `yarn config set yarn-offline-mirror` during the CI setup